go.mod

```diff
@@ -18,4 +20,5 @@ require (
 	github.com/hashicorp/vault/sdk v0.1.14-0.20191017173300-47a54ac8bc6c
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/rainycape/unidecode v0.0.0-20150907023854-cb7f23ec59be // indirect
+	github.com/terraform-providers/terraform-provider-vault v0.0.0-00010101000000-000000000000
```

Comment:
This looks off, maybe being affected by the replace above?

Reply:
As @jasonodonnell found, `main.go` was still using that. I've updated it in 56817de and `go mod tidy` removed that line